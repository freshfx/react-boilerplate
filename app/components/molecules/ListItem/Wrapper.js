import styled from '@emotion/styled'

const Wrapper = styled.li`
  width: 100%;
  height: 3em;
  display: flex;
  align-items: center;
  position: relative;
  border-top: 1px solid #eee;

  &:first-of-type {
    border-top: none;
  }
`

export default Wrapper
