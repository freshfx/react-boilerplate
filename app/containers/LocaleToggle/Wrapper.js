import styled from '@emotion/styled'

const Wrapper = styled.div`
  padding: 2px;
`

export default Wrapper
