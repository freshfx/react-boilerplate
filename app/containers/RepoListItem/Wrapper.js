import styled from '@emotion/styled'

const Wrapper = styled.div`
  width: 100%;
  height: 100%;
  display: flex;
  align-items: space-between;
`

export default Wrapper
