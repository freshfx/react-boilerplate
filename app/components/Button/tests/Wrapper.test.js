import React from 'react'
import {render} from '@testing-library/react'

import Wrapper from '../Wrapper'

describe('Wrapper', () => {
  it('should match snapshot', () => {
    const {container} = render(<Wrapper />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
