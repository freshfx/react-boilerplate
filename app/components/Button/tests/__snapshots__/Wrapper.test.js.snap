// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> should match snapshot 1`] = `
.c0 {
  width: 100%;
  text-align: center;
  margin: 4em 0;
}

<Wrapper>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "Wrapper-sc-5dsnpv-0",
          "isStatic": false,
          "lastClassName": "c0",
          "rules": Array [
            "width:100%;text-align:center;margin:4em 0;",
          ],
        },
        "displayName": "Wrapper",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "Wrapper-sc-5dsnpv-0",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <div
      className="c0"
    />
  </StyledComponent>
</Wrapper>
`;
